README.md

```diff
@@ -2,3 +2,36 @@
 
 The Empirical Software Engineering journal (EMSE) wants to encourage open-science and reproducible research.
 This repository describes the process and infrastructure that support this initiative.
+
+## Open-science Board
+
+Chairs: TBA
+
+Board members: TBA
+
+## Process
+
+To give about this proposal, there are two possibilities:
+* comment on the pull-request discussion tread
+* comment a specific line by clicking in the left margin)
+
+Process description:
+
+1. Once a paper is accepted, the authors have the choice to submit a replication package
+    * upon acceptance, one of the EiCs forwards the paper to the Open-Science Chairs who selects a suitable Open-Science board member.
+    * the acceptance email explicitly asks to submit a URL to the replication package.
+        * email text: "EMSE encourages open-science and reproducible research and your paper can be badged as "open-science". To do so, please send an email containing the URL to the replication package at emsejournal.openscience@chalmers.se. To get more information about the open-science initiative, please visit https://github.com/emsejournal/openscience"
+    * the authors are given 2 weeks to submit their replication package.
```

Comment:
How this is phrased is important. If authors get a feeling there is a risk this will delay the processing and publishing of their paper they might be less inclined to provide a replication package?

Reply:
> How this is phrased is important.

Agree. What change would you propose?